experiment/autobumper/bumper/bumper.go

```diff
@@ -286,6 +286,13 @@ func Run(o *Options) error {
 			}
 		}
 
+		// Check to see if the proper fork exists and if it does not, create one.
+		// TODO(mpherman): Handle case where account has repo of same name as one we are trying to autobump but is not a fork of it (e.g. test-infra)
+		_, err := gc.EnsureFork(o.GitHubLogin, o.GitHubOrg, o.GitHubRepo)
```

Comment:
This _cannot_ be ignored, you must keep track of what this is called if you're going to interact with it.